.github/workflows/yamllint-github-workflows.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: yamllint-github-workflows
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  yamllint-actions:
+    name: yamllint-actions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: install deps
+        run: |
+          sudo apt-get update
+          sudo apt install yamllint
+      - name: run yamllint
+        run: |
+          yamllint -c ./.github/workflows/conf/yamllint.yaml \
```

Comment:
Why not the whole repo and just workflows

Reply:
Long term goal should to have yamllint check for every yaml file; however, currently there are too many warns which makes it a non-trivial task (mostly indent issues). Also, we may want to exclude to auto-generated ones, or at least have a more relaxed yamllint config.

Reply:
@iamniting After doing some investigations, I fixed this action to apply for all yaml files in the repo. Need to use slightly more permissive rules than defaults.

Reply:
excellent